src/course-outline/configure-modal/ConfigureModal.scss

```diff
@@ -5,8 +5,4 @@
   .configure-modal__header {
     padding-top: 1.5rem;
   }
-
-  .configure-modal__body {
-    overflow: visible;
-  }
```

Comment:
Any reason for removing this? It seems to break time dropdown in section configuration.

Reply:
I’ll check on this, with this tule the modal was rendering with a gap in the tabs that have long descriptions and didn’t fit the screen.